src/gui/core/plugin/userui/pom.xml

```diff
@@ -264,4 +272,33 @@
 		</resources>
 
 	</build>
+	
+	<profiles>
+		<profile>
+			<id>disableUsersUI</id>
+			<build>
+				<plugins>
+					<plugin>
+						<artifactId>maven-antrun-plugin</artifactId>
+						<version>1.7</version>
+						<executions>
+							<execution>
+								<phase>compile</phase>
+								<configuration>
+									<tasks>
+										<echo message="Replacing tokens, the output directory is: ${project.build.outputDirectory}" />
+										<replace file="${project.build.outputDirectory}/org/geoserver/geofence/gui/server/service/impl/activateTabs.properties" token="true" value="false" />
+									</tasks>
+								</configuration>
+								<goals>
+									<goal>run</goal>
+								</goals>
+							</execution>
+						</executions>
+					</plugin>
+				</plugins>
+			</build>
+		</profile>
+	</profiles>
+	
 </project>
```

Comment:
well, the formatting looks weird at a first glance but I have double checked and it use the same indentation of the rest of this file...